mne/report.py

```diff
@@ -698,7 +703,13 @@ def _validate_input(self, items, captions, section, comments=None):
         if not isinstance(captions, (list, tuple)):
             captions = [captions]
         if not isinstance(comments, (list, tuple)):
-            comments = [comments]
+            if comments is None:
+                comments = [comments] * len(captions)
+            else:
+                comments = [comments]
+        elif not len(comments) == len(items):
```

Comment:
this looks suspicious / redundant to me. What if `comments='absdfd'` but `len(captions)=1` and `len(items)=5` ?